### DISCUSSION TITLE
'originalFactory.call' is undefined

### DISCUSSION THREAD

**Discussion by tnorlund** [gatsbyjs/gatsby#30748] [Help]
Whenever I develop, I get the error that the 'originalFactory.call' is undefined. Is there a way around this? The site build fine. I believe this has to do with SSR.

<img width="1181" alt="image" src="https://user-images.githubusercontent.com/26452518/113922817-9d9df800-979c-11eb-82fa-7684cb70ff66.png">

**Comment by Isaac-Tait** [comment]
I am getting this same error. It cropped up when I added Algolia search functionality to my site. 

Rather than put the search component into `Layout` I chose to add it to my `blog post list` template... Other than that I followed the [official](https://www.gatsbyjs.com/docs/adding-search-with-algolia/) Gatsby docs to get the bare bones search functionality. 

I came across your Stack Overflow [post](https://stackoverflow.com/questions/66993231/gatsby-no-longer-developing-due-to-originalfactory-call-is-undefined) but I do not think adjusting the `onServiceWorkerUpdateReady` function will resolve for me due to the error cropping up during an Algolia install. Thanks in advance for any guidance you could give to help me solve this... Cheers.

**Comment by alecmolloy** [comment]
also running into this issue, but weirdly it only appears in Safari, not in chrome. seems to be related to my use of the `Auth0Provider` component in my gatsby-browser.tsx file.

**Comment by nishantwrp** [comment]
Was running into this, clearing my browser cache fixed the error.
